core/src/test/java/com/redhat/runtimes/inventory/auth/rbac/RbacParsingTest.java

```diff
@@ -0,0 +1,112 @@
+/* Copyright (C) Red Hat 2023 */
+package com.redhat.runtimes.inventory.auth.rbac;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import org.junit.jupiter.api.Test;
+
+/** Test RBAC parsing. Brought over from notifications-backend and extended */
+class RbacParsingTest {
+
+  private static final ObjectReader RBAC_RAW_READER = new ObjectMapper().readerFor(RbacRaw.class);
+  private static final String BASE = "src/test/resources/rbac-examples/";
+  public static final String ALL = "*";
+
+  @Test
+  void testParseExample() throws Exception {
+    RbacRaw rbac = readValueFromFile("rbac_example.json");
+    assert rbac.data.size() == 2;
```

Comment:
assert is a keyword, not the JUnit 5 methods. Please remove all uses of the assert keyword & replace with appropriate JUnit 5 assertions.

Reply:
Replaced all asserts with JUnit 5 assertions